.github/dependabot.yml

```diff
@@ -118,6 +118,8 @@ updates:
       - dependency-name: com.amazonaws:aws-lambda-java-events
       # Apache Commons
       - dependency-name: commons-io:commons-io
+      # Micrometer
+      - dependency-name: micrometer-io:micrometer-core
     rebase-strategy: disabled
```

Comment:
And you will need to add `micrometer-registry-stackdriver` too as it's also using the property.